client/state/rewind/selectors/get-backup-current-site-size.ts

```diff
@@ -0,0 +1,13 @@
+import type { AppState } from 'calypso/types';
+
+/**
+ * Retrieves current site size in bytes.
+ *
+ * @param state The application state.
+ * @param siteId The site ID for which to retrieve days of backups saved.
+ * @returns The current site size number in bytes.
+ */
+const getBackupCurrentSiteSize = ( state: AppState, siteId: number ): number | undefined =>
+	state.rewind[ siteId ]?.size?.lastBackupSize ?? null;
```

Comment:
```suggestion
	state.rewind[ siteId ]?.size?.lastBackupSize ?? undefined;
```
Maybe return `undefined` as with other selectors, for consistency.